routes/index.js

```diff
@@ -164,12 +164,15 @@ var rootPost = function(req, res, next) {
 
   var ref = (req.headers && req.headers.referer) || false;
   //No POST request allowed from other sources
+  //TODO probably need to fix this for prod docker
   if (ref) {
     var u = url.parse(ref);
     var hostname = u && u.hostname.toLowerCase();
+    var environment = env.toLowerCase();
     if (hostname === "kaha.co" ||
       hostname === "demokaha.herokuapp.com" ||
-      env.toLowerCase() === "stage"
+      environment === "stage" ||
+      environment === "dev"
```

Comment:
Ah. Cause our default env was on `stage`. Dev was being used by only me and
I was doing bulk import. At times I accidentally bulk posted on prod data
so to prevent that I blocked post from Dev and consequently this also
prevents others from entering wrong data on our db if they didn't use the
right environment. Kinda hacky but yeah it was removed for a reason. If you
know any cleaner way to do rid then let me know.
On May 5, 2015 12:55 AM, "anupdhml" <notifications@github.com> wrote:

> In routes/index.js
> <https://github.com/kahaco/kaha/pull/12#discussion_r29645270>:
>
> >      if (hostname === "kaha.co" ||
> >        hostname === "demokaha.herokuapp.com" ||
> > -      env.toLowerCase() === "stage"
> > +      environment === "stage" ||
> > +      environment === "dev"
>
> You removed it here: c79f01c
> <https://github.com/kahaco/kaha/commit/c79f01ca7936c900ec2b3d6729e67234979cecf5>
>
> I added this back on purpose so that POST works on local redis
> installations (with/without docker). Why was it removed in the first place?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kahaco/kaha/pull/12/files#r29645270>.
>